.github/workflows/nightly.yml

```diff
@@ -233,6 +233,67 @@ jobs:
           AWS_SECRET_ACCESS_KEY: "${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}"
         run: make release-s3
 
+  release-cloudsmith:
+    runs-on: ubuntu-latest
+    needs:
+      - build-x86_64-unknown-linux-gnu-packages
+      - build-x86_64-unknown-linux-musl-packages
+      - build-aarch64-unknown-linux-musl-packages
+    steps:
+      - uses: actions/checkout@v1
+      - name: Push amd64 deb
+        id: push-deb-amd64
+        uses: cloudsmith-io/action@master
+        with:
+          api-key: ${{ secrets.CLOUDSMITH_API_KEY }}
+          command: "push"
+          format: "deb"
+          owner: "timber"
+          repo: "vector-nightly"
+          distro: "any-distro"
+          release: "any-version"
```

Comment:
@Hoverbear is this accurate? Aren't we limited by the Glibc version now?

Reply:
For debian/ubuntu we should be  fine. It's a problem for centos7 only.

Reply:
There has been some movement on that front: https://github.com/rust-embedded/cross/issues/455